anomaly_detector/model/som_model.py

```diff
@@ -39,13 +39,15 @@ def train(self, inp, map_size, iterations):
                         bmu_loc = (i, j)
 
             # Update BMU and Neighbours in the map:
-            try:
-                for x in range(-12,12):
-                    for y in range(-12,12):
-                        self.model[bmu_loc[0]+x][bmu_loc[1]+y] = self.model[bmu_loc[0]+x][bmu_loc[1]+y] + (self.alph(iterations, iters)) * \
-                                                                                                         self.neihborhood(np.array(bmu_loc), np.array([bmu_loc[0]+x, bmu_loc[1]+y])) * (current_vector - self.model[bmu_loc[0]+x][bmu_loc[1]+y])
-            except IndexError:
-                pass
+            for x in range(-12, 12):
+                for y in range(-12, 12):
+                    current_x = bmu_loc[0]+x
+                    current_y = bmu_loc[1] +y
```

Comment:
Can you please fix the spaces around `+`, please? (There should by one space before and after)